src/blueprint/from_config/server.rs

```diff
@@ -22,12 +23,13 @@ pub struct Server {
   pub port: u16,
   pub hostname: IpAddr,
   pub vars: BTreeMap<String, String>,
+  #[serde(skip)]
   pub response_headers: HeaderMap,
```

Comment:
We can't skip any field while serializing the blueprint. We would loose out of information this way, and confuse the developer.

Reply:
Consider writing a custom serde function.

Reply:
I used http-serde to serialize the HeaderMap